.travis-tox.ini

```diff
@@ -101,6 +101,7 @@ deps =
     flake8-rst-docstrings
     flake8-comprehensions
     flake8-bugbear;python_version>="3.5"
+    pydocstyle!=4.0.0
```

Comment:
This needs to be ``pydocstyle<=4.0.0`` since the problem isn't in pydocstyle, but in flake8-docstrings https://gitlab.com/pycqa/flake8-docstrings/issues/36